enforcer-rules/src/main/java/com/google/cloud/tools/dependencies/enforcer/LinkageCheckerRule.java

```diff
@@ -116,6 +116,13 @@ public void execute(@Nonnull EnforcerRuleHelper helper) throws EnforcerRuleExcep
         logger.warn("The rule is set to read dependency management section but it is empty.");
       }
 
+      String projectType = project.getArtifact().getType();
+      if (dependencySection == DependencySection.DEPENDENCIES && ! "jar".equals(projectType)) {
+        // When checking non-BOM project, not interested in non JAR artifact (parent pom)
```

Comment:
I still don't think you need "parent pom" here. That is not what this is about, or if it is, that's not what we're checking.

Reply:
Removed "parent pom".